OpenWebAdvocacy/website
# Add script to autogenerate non-translated language files

## Related Issue
Second of three PRs to resolve https://github.com/OpenWebAdvocacy/website/issues/182. One further PR to come for full resolution.

## Summary of Changes
1. [x] Add script to autogenerate non-translated pages for each language
2. [x] Update contributing.md with new language instructions
3. [x] Generated missing non-translated pages for Spanish and Japanese

### Notes
Each page/post/tag included in this PR is a duplicate of the english version with the front matter updated to:
- Update or add the permalink to include the language code
- Add `translated: false` which the final following PR will hook into to display a "not yet translated" banner
- Add layout to blog posts

The main content of the file has not been altered. 

The "not yet translated" banner will be added in a subsequent PR.

### Testing
- View the preview and update the langauge with the toggle to Spanish or Japanese
- Confirm links in the header, footer, blog, and sidebar take you to a page with the same language. You can tell its the same langauge by the language code in the url and the header/footer/sidebar text. The main content of the page will still be in english because this has not been translated.
